MPInspector/InspectorDataSourceProvider.swift

```diff
@@ -20,7 +20,12 @@ class InspectorDataSourceProvider: NSObject, MPListenerProtocol {
      * @param objects is the arguments sent to this api, such as the MPEvent in logEvent
      */
     internal func onAPICalled(_ apiName: String, stackTrace: [Any], isExternal: Bool, objects: [Any]?) {
-        let newData = AllRowData(title: apiName as String, body: stackTrace.description)
+        let stackTraceLine: String = (stackTrace[1] as! String)
+        let trimmedString = stackTraceLine.components(separatedBy: "[")[1].trimmingCharacters(in: .whitespacesAndNewlines)
+        let apiSubstring = trimmedString.components(separatedBy: apiName)[0]
+        let callingAPI = String(apiSubstring).trimmingCharacters(in: .whitespacesAndNewlines)
+        
+        let newData = AllRowData(title: String(format: "%@.%@", arguments: [callingAPI, apiName]) as String, body: stackTrace.description)
```

Comment:
Would it make sense to use more swift-y syntax here and elsewhere? e.g. `"foo: \(foo)"`?

Reply:
totally forgot that Swift let you do this.